docs/data/capture/_index.md

```diff
@@ -291,8 +291,8 @@ The following example captures data from two analog readers that provide a volta
                 "capture_frequency_hz": 1,
                 "name": "rdk:component:board/my-esp32",
                 "additional_params": {
-                  "A2": "",
-                  "A1": ""
+                  "reader_name": "A1",
```

Comment:
There's only be a single reader_name per capture, so this would instead be:


```
{
                "method": "Analogs",
                "capture_frequency_hz": 1,
                "name": "rdk:component:board/my-esp32",
                "additional_params": { "reader_name": "A1" },
},
{
                "method": "Analogs",
                "capture_frequency_hz": 1,
                "name": "rdk:component:board/my-esp32",
                "additional_params": { "reader_name": "A2" },
},
```